simon_says/device_wrapper/include/device_wrapper.hpp

```diff
@@ -0,0 +1,82 @@
+#ifndef SIMON_SAYS_DEVICE_WRAPPER_H
+#define SIMON_SAYS_DEVICE_WRAPPER_H
+
+#include <ros/ros.h>
+#include <geometry_msgs/Twist.h>
+
+#include <simon_says_msgs/Mode.h>
+#include <simon_says_msgs/Status.h>
+
+using namespace simon_says_msgs;
+
+namespace simon_says {
+
+class DeviceWrapper {
+public:
+    explicit DeviceWrapper(ros::NodeHandle& nh, double timeout = 1.0)
```

Comment:
Style is inconsistent: this header file contains all implementations, while device.hpp does not contain any.